projects/tokens/models/sources/base/ethereum_base_sources.yml

```diff
@@ -0,0 +1 @@
+../../../../../models/base_sources/ethereum_base_sources.yml
```

Comment:
when thinking about more sub-projects in the future we can put all the shared base sources in it's separate model folder and add it to the `model-paths:` in every `dbt_project.yml` file.

Reply:
Yeah I was thinking the same, we will basically have shared models and shared macros across projects.